validate/validate.go

```diff
@@ -688,25 +688,7 @@ func (v *Validator) CheckAnnotations() (errs error) {
 
 // CapValid checks whether a capability is valid
 func CapValid(c string, hostSpecific bool) error {
-	isValid := false
-
-	if !strings.HasPrefix(c, "CAP_") {
-		return fmt.Errorf("capability %s must start with CAP_", c)
-	}
-	for _, cap := range capability.List() {
-		if c == fmt.Sprintf("CAP_%s", strings.ToUpper(cap.String())) {
-			if hostSpecific && cap > LastCap() {
-				return fmt.Errorf("%s is not supported on the current host", c)
-			}
-			isValid = true
-			break
-		}
-	}
-
-	if !isValid {
-		return fmt.Errorf("invalid capability: %s", c)
-	}
-	return nil
+	return capsCheck.CapValid(c, hostSpecific)
```

Comment:
A simpler solution is this:
```
// CapValid checks whether a capability is valid.
//
// DEPRECATED: Use [github.com/opencontainers/runtime-tools/validate/capabilities.CapValid] directly.
var CapValid = capsCheck.CapValid
```

Whether the alias should be marked as deprecated is open for debate, just as whether this alias is needed at all.

I think it makes sense, but I don't know how this package manages API stability.

Reply:
True. Let's hear from the maintainers which style they prefer.

Reply:
I understand the concern that since this is a variable, it can be changed, but I guess we can live with it, since this stuff is deprecated and to be removed soon.